src/main/java/stsjorbsmod/relics/GrimoireRelic.java

```diff
@@ -37,7 +48,55 @@ public void onVictory() {
         AbstractPlayer p = AbstractDungeon.player;
 
         if (p.currentHealth > 0) {
-            p.heal(MemoryManager.forPlayer(p).countCurrentClarities() * HEAL_PER_CLARITY);
+            int healAmount = MemoryManager.forPlayer(p).countCurrentClarities() * HEAL_PER_CLARITY;
+            p.heal(healAmount);
+            stats.merge(STAT_AMOUNT_HEALED, healAmount, Math::addExact);
+        }
+    }
+
+    @Override
+    public AbstractRelic makeCopy() {
+        GrimoireRelic copy = (GrimoireRelic) super.makeCopy();
+        copy.stats = stats;
+        return copy;
+    }
+
+    @Override
+    public String getStatsDescription() {
+        return String.format(STAT_AMOUNT_HEALED, stats.get(STAT_AMOUNT_HEALED));
+    }
+
+    @Override
+    public String getExtendedStatsDescription(int totalCombats, int totalTurns) {
+        float amountHealed = (float) stats.get(STAT_AMOUNT_HEALED);
+        return new StringBuilder()
+                .append(getStatsDescription())
+                .append(String.format(STAT_PER_TURN, STATS_FORMAT.format(amountHealed / Math.max(totalTurns, 1))))
```

Comment:
Seeing some oddities with the `totalTurns` and `totalCombats` counts. They're far higher than I expected and I think they're skewing the stats.

Reply:
I'm not sure what I can do about totalTurns and totalCombats here. These are coming from the stats mod.

Reply:
No problem then, just worth mentioning.